src/components/CommentsToggle.js

```diff
@@ -27,6 +38,19 @@ export const CommentsToggle = ({ review_id, comment_count }) => {
     }
   }, [collapsed]);
 
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setIsLoading(true);
+    postComment(review_id, username, input).then((data) => {
+      setIsLoading(false);
+      setComments((currComments) => {
+        return [...currComments, data.comment[0]];
+      });
```

Comment:
This is where the catch would happen - you can catch the error and display a little error message/set Error state